pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -941,6 +941,13 @@ func handleAppNetworkCreate(ctx *zedrouterContext, key string,
 	}
 	publishAppNetworkStatus(ctx, &status)
 
+	if validateAppNetworkConfig(ctx, config, &status) {
```

Comment:
Nit but I'd assume that a function called validate* would return true to indicate success and false for failure. This is the reverse.

Reply:
sure. implemented.